tests/e2e/specs/activation-flow.spec.js

```diff
@@ -34,4 +59,27 @@ describe( 'Activation flow', () => {
 		await waitForWpAdmin();
 		expect( await page.$( '#message.error' ) ).toBe( null );
 	} );
+
+	it( 'Should display all admin sections', async () => {
+		await visitAdminPage( '/options-general.php', '?page=parsely' );
+
+		await page.waitForXPath( '//h2[contains(text(), "Basic Settings")]' );
+		expect( await checkH2DoesNotExist( 'Requires Recrawl Settings' ) ).toBe( true );
+		expect( await checkH2DoesNotExist( 'Advanced Settings' ) ).toBe( true );
+
+		await selectScreenOptions( true, true );
+
+		await page.waitForXPath( '//h2[contains(text(), "Basic Settings")]' );
+		await page.waitForXPath( '//h2[contains(text(), "Requires Recrawl Settings")]' );
+		await page.waitForXPath( '//h2[contains(text(), "Advanced Settings")]' );
+
+		await selectScreenOptions( false, true );
```

Comment:
What would you do if we had 6 or 10 toggleable sections, instead of 2? Probably better to treat this argument as a single associated array (section-name => true/false), rather than multiple boolean arguments.